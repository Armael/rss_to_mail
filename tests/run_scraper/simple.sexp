("#content > ul > li"
 (T
  (entry
   (R ("a"
    (T title link))))))
