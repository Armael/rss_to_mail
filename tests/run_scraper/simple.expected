Feed title: None
Feed link: None
Feed icon: None
Entries (3):
	id: None
	title: C
	link: c
	summary: None
	id: None
	title: B
	link: b
	summary: None
	id: None
	title: A
	link: a
	summary: None
